### PR TITLE
added node.max_local_storage_nodes option for Elasticsearch

### PR DESCRIPTION
**Pull request by JinIgarashi**
added `node.max_local_storage_nodes` option into command line in order to share the same data directory with different photon instance.

https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#max-local-storage-nodes

### CODE REVIEW

File: .docker/entrypoint.sh [hunk 1]
```diff
@@ -1,22 +1,26 @@
 #!/bin/sh
 
+if [ ! -n "$MAX_LOCAL_STORAGE_NODES" ]; then
+  MAX_LOCAL_STORAGE_NODES = 1
```
Comment (kenseii) [inline .docker/entrypoint.sh:4]: I think this doesnt set the storage node if MAX_LOCAL_STORAGE_NODES= is not set inside .env.

when run with MAX_LOCAL_STORAGE_NODES not having a value inside .env, the build returns this error
```
Attaching to photon_photon_1
photon_1  | /photon/entrypoint.sh: line 4: MAX_LOCAL_STORAGE_NODES: not found
photon_1  | Creating search index
photon_1  | 2021-03-29 04:39:41,202 [main] WARN  de.komoot.photon.App - could not start photon: "-max-local-storage-nodes": couldn't convert "-nominatim-import" to an integer
```

**Review by kenseii — CHANGES_REQUESTED** [review]
Not sure if it is necessary to fix this validation but we can consider it.

**Comment by JinIgarashi** [comment]
@kenseii I fixed entrypoint.sh
https://github.com/MIERUNE/photon/pull/5#discussion_r603011586

**Comment by kenseii** [comment]
LGTM! thank you

**Pull request merged**